.buildkite/pipelines/chrome_forward_testing.yml

```diff
@@ -0,0 +1,355 @@
+env:
```

Comment:
Was this pipeline intended?

Reply:
You're right, it wasn't.
I believe it appeared through conflict resolution. Maybe keeping the missing file would have been the right resolution, this pipeline file won't be used in `non-main` branches. 

However, if `8.x` and `main` diverge significantly, we might want to run the forward testing for the two branches separately. `8.x` is going to live for a while, no? What do you think?